lib/command.php

```diff
@@ -51,6 +52,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->write($result);
         }
 
+        if ($errorOutput !== '') {
+            $stdErr->write($errorOutput);
```

Comment:
if everything works according to plan, this variable should always be empty.

but lets output it, just to be sure